3scale/porta
# Updates Babel/Webpack packages and cleans up babel a bit

**What this PR does / why we need it**:

It unifies Babel configuration used by webpack and cleans up `.babelrc`, removing deprecated plugins and presets and installing up-to-date versions.

It also updates webpacker, babel, eslint and other related stuff.

**Verification steps** 
1. Install dependencies:
    * `npm i && bundle install`

2. Verify Babel, Webpack, Karma, Jest, Jasmine, React... all work together (no errors are thrown running the following commands):
    * `bundle exec rake webpack:compile`
    * `npm run test`
    * `npm run jest`
